src/DynamoSandbox/DynamoSandbox.csproj

```diff
@@ -23,13 +23,15 @@
   <PropertyGroup>
     <ApplicationIcon>logo_square_32x32.ico</ApplicationIcon>
   </PropertyGroup>
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Release|AnyCPU'">
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Release|x64'">
     <UseVSHostingProcess>false</UseVSHostingProcess>
     <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|AnyCPU'">
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|x64'">
     <Prefer32Bit>false</Prefer32Bit>
     <UseVSHostingProcess>true</UseVSHostingProcess>
+    <OutputPath>..\..\bin\AnyCPU\Debug\</OutputPath>
+    <PlatformTarget>AnyCPU</PlatformTarget>
```

Comment:
Should the PlatformTarget be AnyCPU or it should be x64?? The OutputPath shouldn't be hardcoded here.